core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -145,22 +142,34 @@ public void removeNetworks(String identifier) {
     }
 
     private void removeNetwork(String networkName) {
-        List<Network> networks;
         try {
-            networks = dockerClient.listNetworksCmd().withNameFilter(networkName).exec();
-        } catch (DockerException e) {
-            LOGGER.trace("Error encountered when looking up network for removal (name: {}) - it may not have been removed", networkName);
-            return;
-        }
+            try {
+                // First try to remove by name
+                dockerClient.removeNetworkCmd(networkName).exec();
+            } catch (Exception e) {
+                LOGGER.trace("Error encountered removing network by name ({}) - it may not have been removed", networkName);
```

Comment:
Two tests running with same network name will fail , at least the second create will. They will not understand why since the failure is hidden under trace level. I would keep this at error level or warn.